src/System.Net.Sockets/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -34,14 +35,34 @@ public void EventSource_EventsRaisedAsExpected()
                     var events = new ConcurrentQueue<EventWrittenEventArgs>();
                     listener.RunWithCallback(events.Enqueue, () =>
                     {
-                        // Invoke a test that'll cause some events to be generated
+                        // Invoke several tests to execute code paths while tracing is enabled
```

Comment:
When we do semi-advanced Task stuff, like calling GetAwaiter, can we add a comment about why we are doing this instead of something like await or Task.Wait or Task.Result etc?